frontend/src/api/tests/UserManagementApi.test.js

```diff
@@ -0,0 +1,42 @@
+import axios from "axios";
+import Enzyme from "enzyme";
+import Adapter from "enzyme-adapter-react-16";
+import { describe, expect, jest, it } from "@jest/globals";
+import * as authenticationUtil from "../AuthenticationUtil";
+import * as UserManagementApi from "../UserManagementApi";
+
+Enzyme.configure({ adapter: new Adapter() });
+
+jest.mock("axios");
+jest.mock("../AuthenticationUtil");
+
+const authorizationHeader = {
+  headers: {
+    Authorization: "Bearer the_token"
+  }
+};
+
+describe("UserManagementApi", () => {
+  describe("createUser()", () => {
+    it("should call axios.post and return a 200", () => {
```

Comment:
You are not checking if it returns 200, or the return value at all

1) Rename test: `Should call axios.post() with expected arguments`
2) Add a test for the return value (mock post to return something)

Reply:
I don't think the axios post returns anything at all for the createUser() api. It just sends a post request to the server and that's it. So I'm not sure if a mocked return value is required at all?